.github/workflows/cd.yml

```diff
@@ -0,0 +1,128 @@
+name: CD
+
+on:
+  release:
+    types: [published]
+  schedule:
+    - cron: '0 0 * * 0' # Every Sunday at midnight UTC
+  workflow_dispatch:
```

Comment:
Running the workflow on a schedule allows one to verify that the release builds are passing in times of inactivity on the repository or for any other needs. The manual trigger allows triggering it at will, for debugging purposes. The `release` job only responds to the release event.